cmd/ps_test.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestScaleFail(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("", "tmpdir")
+	if err != nil {
+		t.Fatalf("error creating temp directory (%s)", err)
+	}
+	err = os.Mkdir(tmpDir+"/.deis", 0666)
+	if err != nil {
+		t.Fatalf("error creating temp directory (%s)", err)
+	}
+	os.Setenv("DEIS_PROFILE", "testing")
+	os.Setenv("HOME", tmpDir)
```

Comment:
This will break on windows, cause home is different. I wrote `settings.SetHome` to handle the cross platform differences transparently.